src/lib/table/table.md

```diff
@@ -45,3 +45,9 @@ an update via the table's data source.
 In the near future, we will provide a simplified version of the data-table with an easy-to-use
 interface, material styling, array input, and more out-of-the-box features (sorting, pagination,
 and selection).
+
+### Accessibility
+Tables without text or labels should be given a meaningful label via `aria-label` or
+`aria-labelledby`.
```

Comment:
We should mention `aria-readonly` regardless of what we do with it.

(it defaults to `true` if it's not set, but most people probably make readonly tables)